### PR TITLE
Add resizeObserver to trigger popper update #270

### LINKED ISSUE #270: auro-select issue:  select dropdown position is off
Reference auro-select issue
AlaskaAirlines/auro-select/issues/225

### PR DESCRIPTION
**Pull request by rmenner**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Enhancements:
- Integrate ResizeObserver to monitor element size changes and trigger popper updates accordingly.

### CODE REVIEW

File: src/auro-dropdown.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export class AuroDropdown extends LitElement {
     this.isPopoverVisible = false;
     this.matchWidth = false;
     this.noHideOnThisFocusLoss = false;
+    this.resizeObserver = null;
```
Comment (jordanjones243) [inline src/auro-dropdown.js]: Should this be `null` or `undefined`?

File: src/auro-dropdown.js [hunk 2]
```diff
@@ -365,6 +368,7 @@ export class AuroDropdown extends LitElement {
       this.triggerChevron.removeAttribute('data-expanded');
     }
     this.dispatchEventDropdownToggle();
+    this.resizeObserver.unobserve(this.trigger);
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/auro-dropdown.js]: Should add checks like Sourcery suggests.

**Review by jordanjones243 — CHANGES_REQUESTED** [review]

File: src/auro-dropdown.js (hunk 1 above)
Comment (rmenner) [reply src/auro-dropdown.js]: Undefined is the right call (according to a quick search)

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**